### PR TITLE
HV 4.5

### PR DESCRIPTION
**Pull request by ramijebara**
Please review the proposed release notes for HV 4.5

### CODE REVIEW

File: documentation/release-notes/topics/version_4.rst [hunk 1]
```diff
@@ -2,6 +2,98 @@
...
+============
+- **AS-15227** The All location carbon footprint report is accessible by All users if the Reporting license is enabled.
+
+=================
+Notable bug fixes
+=================
+- **AS-14510** Fixed a bug that caused some invalid email address formats to be accepted as valid.
+- **AS-14543** Fixed a bug that made it not possible to delete a numeric custom property if a default value is not set.
+- **AS-14598** Fixed a bug that caused an error message to not be displayed after attempting to link a manual sensor to an asset that already has a sensor of the same type.
+- **AS-14568** Fixed a bug that caused the Camera asset type to not be selectable in Advanced Search filters.
```
Comment (tfadler) [inline documentation/release-notes/topics/version_4.rst:95]: May want to include
AS-14532

Comment (ramijebara) [reply documentation/release-notes/topics/version_4.rst:95]: Added. Thanks!

File: documentation/release-notes/topics/version_4.rst [hunk 2]
```diff
@@ -2,6 +2,99 @@
...
+   3. Microsoft has `announced <https://devblogs.microsoft.com/microsoft365dev/retirement-of-office-365-connectors-within-microsoft-teams/>`_ the sudden and quick retirement of the Webhook interface within MSTeams. This impacts the Notifications Channels functionality. We are evaluating options and will be addressing this in an upcoming release.
+
+===========================================================
+New Licensed Feature - Reporting/Carbon footprint reporting
+===========================================================
+This release introduces a Reports section into the Hyperview application. The first report we are introducing is the Carbon footprint report. With this release, you will be able to track a Location's carbon footprint. Track the carbon footprint of Rack workloads and assess the impact of the average offset percentage on your carbon footprint in aggregate and per location.
+
+To support this report report:
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:20]: double word: "report report"

File: documentation/release-notes/topics/version_4.rst [hunk 3]
```diff
@@ -2,6 +2,99 @@
...
+
+=======================
+New Feature - Dark mode
+=======================
+This release introduces a dark-mode GUI theme. The Hyperview GUI will by default set the theme automatically based on the user system preferences. The theme can also be specifically set by the user from the **Account -> Appearance** menu.
+
+================================
+New Feature - In-app Help bubble
+================================
+With this release, we are introducing a new in-app help system powered by `Userflow <https://userflow.com/>`_. Where applicable, We are planning a phased roll-out to customers.
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:37]: "Where applicable, We are planning...", should be: "Where applicable, we are planning..."

File: documentation/release-notes/topics/version_4.rst [hunk 4]
```diff
@@ -2,6 +2,99 @@
...
+   2. Support for the *Windows* version of the Data Collector will end on *January 31, 2025*. Customers are encouraged to switch to the Linux version before then. Detailed :ref:`installation instructions <Setting-up-data-collectors-doc>` are in the product documentation.
+   3. Microsoft has `announced <https://devblogs.microsoft.com/microsoft365dev/retirement-of-office-365-connectors-within-microsoft-teams/>`_ the sudden and quick retirement of the Webhook interface within MSTeams. This impacts the Notifications Channels functionality. We are evaluating options and will be addressing this in an upcoming release.
+
+===========================================================
+New Licensed Feature - Reporting/Carbon footprint reporting
+===========================================================
+This release introduces a Reports section into the Hyperview application. The first report we are introducing is the Carbon footprint report. With this release, you will be able to track a Location's carbon footprint. Track the carbon footprint of Rack workloads and assess the impact of the average offset percentage on your carbon footprint in aggregate and per location.
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:18]: "Carbon footprint report" should be "Carbon Footprint report" or "carbon Footprint report".

Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:18]: "Location's" should probably be "location's"

"Rack workloads" should probably be "rack workloads".

File: documentation/release-notes/topics/version_4.rst [hunk 5]
```diff
@@ -2,6 +2,99 @@
...
+
+To support this report report:
+- Workload type has been added as a standard property for Racks.
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:21]: "Workload type" should be "Workload Type"
"Racks" should be "racks"

File: documentation/release-notes/topics/version_4.rst [hunk 6]
```diff
@@ -2,6 +2,99 @@
...
+- Workload type is also settable in bulk.
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:22]: "Workload type" should be "Workload Type"

File: documentation/release-notes/topics/version_4.rst [hunk 7]
```diff
@@ -2,6 +2,99 @@
...
+
+More enhancements and reports are planned in the coming releases of Hyperview this year and beyond.
+
+.. note:: Please contact the Hyperview sales or support teams for more information.
+
+=======================
+New Feature - Dark mode
+=======================
+This release introduces a dark-mode GUI theme. The Hyperview GUI will by default set the theme automatically based on the user system preferences. The theme can also be specifically set by the user from the **Account -> Appearance** menu.
+
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:33]: "dark-mode" should be "dark mode"

File: documentation/release-notes/topics/version_4.rst [hunk 8]
```diff
@@ -2,6 +2,99 @@
...
+With this release, we are introducing a new in-app help system powered by `Userflow <https://userflow.com/>`_. Where applicable, We are planning a phased roll-out to customers.
+
+===================================
+Enhanced Feature: Business Entities
+===================================
+Business Entity has been added as an optional column in the following grids:
+
+1. Location -> Information -> Assets
+2. Assets by type
+3. Connectivity -> Connection
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:46]: "Connection" should be "Connections"

File: documentation/release-notes/topics/version_4.rst [hunk 9]
```diff
@@ -2,6 +2,99 @@
...
+==========================================
+Enhanced Licensed Feature: Firmware update
+==========================================
+- Support for the Panduit Gen6 rack PDU products has been added.
+
+======================================
+Other notable changes and improvements
+======================================
+- USB has been added to the list of port types.
+- The hostname property can now be deleted.
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:80]: "hostname" should be "Host Name"

File: documentation/release-notes/topics/version_4.rst [hunk 10]
```diff
@@ -2,6 +2,99 @@
...
+- Discovery state is now available in Advanced Search filters.
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:81]: "Discovery state" should be "Discovery State"

File: documentation/release-notes/topics/version_4.rst [hunk 11]
```diff
@@ -2,6 +2,99 @@
...
+- Temperature and Humidity can now be added to manual sensors.
+
+============
+Known issues
+============
+- **AS-15227** The All location carbon footprint report is accessible by All users if the Reporting license is enabled.
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:87]: "All users..." should be "all users..."

File: documentation/release-notes/topics/version_4.rst [hunk 12]
```diff
@@ -2,6 +2,99 @@
...
+
+.. note:: Please contact the Hyperview sales or support teams for more information.
+
+=======================
+New Feature - Dark mode
+=======================
+This release introduces a dark-mode GUI theme. The Hyperview GUI will by default set the theme automatically based on the user system preferences. The theme can also be specifically set by the user from the **Account -> Appearance** menu.
+
+================================
+New Feature - In-app Help bubble
```
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:35]: "In-app Help bubble" should be "In-app help bubble"

File: documentation/release-notes/topics/version_4.rst (hunk 5 above)
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:21]: The Workload Type property has been added to all asset types.

File: documentation/release-notes/topics/version_4.rst (hunk 2 above)
Comment (kyledickau) [inline documentation/release-notes/topics/version_4.rst:20]: We also added a PUE Setting property

**Review by kyledickau — APPROVED** [review]

File: documentation/release-notes/topics/version_4.rst [hunk 13]
```diff
@@ -2,6 +2,99 @@
...
 
+*******************
+Hyperview 4.5 (TBD)
+*******************
+This section covers significant changes and bug fixes in Hyperview 4.5.x since version 4.4.x
+
+.. important::
+   1. This release has changes and improvements to the Data Collector software. We strongly recommend updating your installed Data Collectors to the latest version to maintain an optimal monitoring and discovery experience.
+   2. Support for the *Windows* version of the Data Collector will end on *January 31, 2025*. Customers are encouraged to switch to the Linux version before then. Detailed :ref:`installation instructions <Setting-up-data-collectors-doc>` are in the product documentation.
+   3. Microsoft has `announced <https://devblogs.microsoft.com/microsoft365dev/retirement-of-office-365-connectors-within-microsoft-teams/>`_ the sudden and quick retirement of the Webhook interface within MSTeams. This impacts the Notifications Channels functionality. We are evaluating options and will be addressing this in an upcoming release.
```
Comment (Thomathan) [inline documentation/release-notes/topics/version_4.rst:13]: MSTeams should be Microsoft Teams

Comment (ramijebara) [reply documentation/release-notes/topics/version_4.rst]: Changed.

**Review by Thomathan — APPROVED** [review]

**Pull request merged**